### ISSUE TITLE
A forward facing display that would only show the text output of Dasher in an adjustable font, size, and color.

### ISSUE THREAD

**Issue by ipomoena** [ipomoena/dasher#10]
When I am in a bar on other loud environment, my speakers are worthless and anyone I have a conversation with has to stand behind me where I can't see them. :(

**Comment by ipomoena** [comment]
One way to do it is allow extra top-level window synchronized with dasher editor.
This window could be maximize this window on second display.
There are reasonably priced small displays suitable for wheelchair 
i.e. http://www.adafruit.com/categories/506

**Comment by willwade** [comment]
I realise this is outside the realm of dasher but.. another way is to somehow have a software bluetooth HID solution on the dasher computer - then a third party device (iPhone, Android whatever..) reads the output. Of course doesn't have to be bluetooth (e.g. [TCP/IP](https://github.com/willwade/MacroServerMac/tree/master/MacroServer#readme), [Sound](https://www.chirp.io)) but everything else relies on a dedicated solution on the receiving end.. Then again maybe thats a good idea since Dasher doesn't input into any old text area..(and its available on most platforms)

**Comment by smoothsaling** [comment]
A friend of mine added code to Dasher to port the text box to an external monitor. I use a 7" USB powered screen and it works great. Working on some details but should have it to share in a week or 2. It is pretty badass.

**Comment by smoothsaling** [comment]
@jcope did this. It works beautifully.

**Comment by smoothsaling** [comment]
@jcope have you submitted your code for the forward facing display back to the public code? Do we need to refine it first?

**Comment by jcope** [comment]
@smoothsaling Yes, the code did make it back up into @ipomoena 's branch, pull request numbers 111-114 (we had trouble squashing and merging!) Should be in there and all set to go!

**Comment by willwade** [comment]
Any chance of a small write up of how it’s put together (hardware wise) and configured. I’ll put this into the website docs. Thanks!

**Comment by smoothsaling** [comment]
I'll let @jcope give you the specs but here are a couple of pictures.
![Uploading Chair copy.jpg…]()
![computer back copy](https://user-images.githubusercontent.com/14254466/37693254-c0c5cd60-2c94-11e8-8442-cae0d7d7b33d.jpg)

**Comment by smoothsaling** [comment]
![chair copy](https://user-images.githubusercontent.com/14254466/37693286-f69e430e-2c94-11e8-8a75-8dbbd501d951.jpg)

**Comment by smoothsaling** [comment]
The display has be USB powered. I use this one:
https://www.mimomonitors.com/collections/vesa-mount/products/copy-of-mimo-um-760f-7-display

The mount I use is the following:
https://www.amazon.com/gp/product/B002WSZ0MQ

I will let @jcope add more but his brother was responsible for the mounting.
